crates/router/src/connector/checkout/transformers.rs

```diff
@@ -1220,13 +1270,22 @@ pub enum CheckoutDisputeTransactionType {
 impl From<CheckoutWebhookEventType> for api::IncomingWebhookEvent {
     fn from(transaction_type: CheckoutWebhookEventType) -> Self {
         match transaction_type {
-            CheckoutWebhookEventType::AuthenticationStarted => Self::EventNotSupported,
-            CheckoutWebhookEventType::AuthenticationApproved => Self::EventNotSupported,
+            CheckoutWebhookEventType::AuthenticationStarted
+            | CheckoutWebhookEventType::AuthenticationApproved
+            | CheckoutWebhookEventType::AuthenticationAttempted => Self::EventNotSupported,
+            CheckoutWebhookEventType::AuthenticationExpired
+            | CheckoutWebhookEventType::AuthenticationFailed
+            | CheckoutWebhookEventType::PaymentAuthenticationFailed => {
+                Self::PaymentIntentAuthorizationFailure
+            }
             CheckoutWebhookEventType::PaymentApproved => Self::EventNotSupported,
             CheckoutWebhookEventType::PaymentCaptured => Self::PaymentIntentSuccess,
             CheckoutWebhookEventType::PaymentDeclined => Self::PaymentIntentFailure,
             CheckoutWebhookEventType::PaymentRefunded => Self::RefundSuccess,
             CheckoutWebhookEventType::PaymentRefundDeclined => Self::RefundFailure,
+            CheckoutWebhookEventType::PaymentCanceled => Self::PaymentIntentCancelled,
```

Comment:
When do we get this Canceled status? if this is during authentication then should be mapped to payment failure

Reply:
Occurs when the customer has canceled the payment via a provider's platform.